packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -38,3 +50,15 @@ export interface UTXO {
   txid: string
   vout: number
 }
+
+export interface GetCollateralToken {
+  token: string
+  height: number
+}
+
+export interface CollateralToken {
```

Comment:
Maybe `CollateralTokenResult` to know that it's an interface returned from a call